### ISSUE TITLE
Version 1.14.30 doesn't find my compiler in the PATH with a specific toolchain

### ISSUE THREAD

**Issue by Tuxbee** [microsoft/vscode-cmake-tools#3127]
### Brief Issue Summary

Given this toolchain file : 

```
set(CMAKE_SYSTEM_NAME Generic)
set(CMAKE_SYSTEM_VERSION 0.1)
set(CMAKE_SYSTEM_PROCESSOR MSP)
message(STATUS "Setting C/C++ compiler to MSP430 GCC")
set(CMAKE_C_COMPILER msp430-gcc CACHE INTERNAL "")
set(CMAKE_CXX_COMPILER msp430-gcc CACHE INTERNAL "")
```
If I use version 1.13.45 it works (the configuration works).
If I use version 1.14.30, I got the following error : 
[cmake]   The CMAKE_C_COMPILER:
[cmake] 
[cmake]     msp430-gcc
[cmake] 
[cmake]   is not a full path and was not found in the PATH.

### Additional Information

Here is the kit definition : 
 {
    "name": "GCC 4.6.3 msp430 - MSP",
    "toolchainFile": "PATH\\TO\\msp.cmake"
  }

**Comment by gcampbell-msft** [comment]
Might be somewhat related: #3125

**Comment by benmcmorran** [comment]
Thanks for the report! Are you using Windows, and is `msp430-gcc` on a path that includes "mingw"? If so, it's likely that #3138 will fix this issue.

**Comment by benmcmorran** [comment]
A potential fix for this issue from #3138 is now publicly available in pre-release channel extension version 1.15.3. Let us know if you're still seeing this issue in the pre-release version.

**Comment by Tuxbee** [comment]
Hello,
I can confirm that the issue is solved in version 1.15.4.
Thank you for the help.

**Comment by benmcmorran** [comment]
Thanks for confirming. This fix is also included in version 1.14.31, which is not a pre-release and was just released to the marketplace.
